workflows/methylseq/main.nf

```diff
@@ -199,6 +213,10 @@ workflow METHYLSEQ {
     )
 
     emit:
+    bam            = ch_bam                      // channel: [ path(bam) ]
```

Comment:
// channel:  [ val(meta), path(bam) ]

and same applies to others, perhaps?

Reply:
will update in next. Thanks